### ISSUE TITLE
Xcode project generator

### ISSUE THREAD

**Issue by jimon** [beardsvibe/buildfox#93]
We should have basic Xcode support

**Comment by den-mentiei** [comment]
Should we generate a project with filters + ninja build step?

**Comment by jimon** [comment]
Yes, filters and ninja build step only, exactly the same with MSVS project :)

**Comment by jimon** [comment]
Probably it's possible to use https://github.com/kronenthaler/mod-pbxproj (use it as dependency or just fork it), though it's unable to create new projects.
Meanwhile I was able to create external build system Xcode project, use ninja with it and debug an executable.

**Comment by jimon** [comment]
Most of work is possible to do through mod-pbxproj or we can just create our own generator.

The only issue is setting executable name : it isn't set in .pbxproj but in .xcscheme which is user configuration folder ... kinda similar story with Qt Creator. Probably first iteration will require user to manually set executable path :(

**Comment by jimon** [comment]
Hey, another issue - how to we call something else of Project -> Clean ? For ninja we need to call `ninja -t clean`, but XCode is calling something else :/
